gateway-service/src/main/java/org/zowe/apiml/gateway/filters/PageRedirectionFilterFactory.java

```diff
@@ -185,6 +189,12 @@ private Mono<Void> processNewLocationUrl(ServerWebExchange exchange, Config conf
         return Mono.empty();
     }
 
+    boolean isGateway(Optional<ServiceInstance> targetInstance) {
+        return targetInstance.isPresent() &&
+            StringUtils.isNoneEmpty(targetInstance.get().getServiceId()) &&
+            targetInstance.get().getServiceId().equalsIgnoreCase(CoreService.GATEWAY.getServiceId());
```

Comment:
```suggestion
        return targetInstance.filter(target -> CoreService.GATEWAY.getServiceId().equalsIgnoreCase(target.getServiceId()))
        .orElse(false);
```

Reply:
do you mean? return targetInstance.filter(target -> CoreService.GATEWAY.getServiceId().equalsIgnoreCase(target.getServiceId()))
            .isPresent();

Reply:
yes, sorry, I wrote it on the fly